datastore/save.go

```diff
@@ -67,8 +67,13 @@ func reflectFieldSave(props *[]Property, p Property, name string, opts saveOpts,
 		case reflect.Interface:
 			// Extract the interface's underlying value and then retry the save.
 			// See issue https://github.com/googleapis/google-cloud-go/issues/1474.
-			v = v.Elem()
-			return reflectFieldSave(props, p, name, opts, v)
+			if v.IsNil() {
+				// Nil pointer becomes a nil property value (unless omitempty, handled above).
```

Comment:
Where is omitEmpty handled? Comment could be clearer.

Reply:
Agree. Probably, "embracing call" will sound better